features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/index.jsx

```diff
@@ -361,6 +363,15 @@ class Details extends React.Component {
         this.setState({ api });
     }
 
+    isAsyncAPI(api) {
+        if (api) {
+            if (api.type === 'WS' || api.type === 'WEBSUB' || api.type === 'SSE') {
```

Comment:
You can simply return (api.type === 'WS' || api.type === 'WEBSUB' || api.type === 'SSE')

Reply:
done